feature-libs/product-configurator/common/components/configure-product/configure-product.component.ts

```diff
@@ -30,8 +30,36 @@ export class ConfigureProductComponent {
 
   ownerTypeProduct: CommonConfigurator.OwnerType =
     CommonConfigurator.OwnerType.PRODUCT;
-
+  /**
+   * Prevents page down behaviour when users press space key to select buttons
+   *
+   * @param {KeyboardEvent} event - Keyboard event
+   * @param {Configurator.Group} group - Entered group
+   */
+  preventScrollingOnSpace(event: KeyboardEvent): void {
+    if (event.code === 'Space') {
+      event.preventDefault();
+    }
+  }
+  /**
+   * Fired on keyboard events, checks for 'enter' or 'space' and naviagtes users to the configurator
+   *
+   * @param {KeyboardEvent} event - Keyboard event
+   * @param {Configurator.Group} group - Entered group
```

Comment:
The second parameter is a **product** and not a **group**. Please, correct it

Reply:
Done